bin/webpack.js

```diff
@@ -83,8 +83,18 @@ if (!cli.installed) {
 	console.error(notify);
 
 	const isYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
+	const isPnpm = fs.existsSync(path.resolve(process.cwd(), "pnpm-lock.yaml"));
```

Comment:
Let's avoid multiple fs call, it reduce performance, let's do it:
```js
const isNpm = fs.existsSync(path.resolve(process.cwd(), "package-lock.json"));

let isYarn = false;

if (!isNpm) {
  isYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
}

let isPnpm = false;

if (!isYarn) { 
  isPnpm = fs.existsSync(path.resolve(process.cwd(), "pnpm-lock.yaml"))
}

```

npm is most popular, then `yarn`, then `pnpm`, so we avoid extra fs calls for `npm`

Reply:
Let's improve it in `webpack-cli` too.